src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -369,8 +378,13 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
                 <search.Component model={search} />
               </>
             )}
-            {!loading && fields.length > 1 && (
-              <FieldSelector label="Field" options={fields} value={value} onChange={model.onFieldSelectorChange} />
+            {!loading && options.length > 1 && (
```

Comment:
Btw, do you know why we do `options.length > 1` not `options.length > 0`?

Reply:
Because the ALL value is always there